CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 
 cmake_minimum_required(VERSION 3.13)
 
-project(sdbus-c++ VERSION 1.2.0 LANGUAGES C CXX)
+project(sdbus-c++ VERSION 2.0.0 LANGUAGES C CXX)
```

Comment:
I did this, because the library still gets created as `libsdbus-c++.so.1.2.0` which is not correct